Formula/instantclient-tools.rb

```diff
@@ -15,6 +15,15 @@ class InstantclientTools < Formula
   sha256 "dfd2d9a2721d2e2a90d8053a8e4c9d0c9a68e8d47c9c99e1b80e8fa1c2edb99c"
 
   def install
-    lib.install Dir["*.dylib*"]
+    if HOMEBREW_PREFIX.to_s != "/usr/local"
+      system DevelopmentTools.locate("install_name_tool"), "-add_rpath", HOMEBREW_PREFIX/"lib", "tools"
+    end
+    lib.install Dir["*.dylib"]
+    bin.install ["impdp"]
+    bin.install ["expdp"]
+    bin.install ["exp"]
+    bin.install ["imp"]
+    bin.install ["wrc"]
+    bin.install ["sqlldr"]
```

Comment:
Is there a reason you chose to list them separately rather than a single array? Also, `brew audit --strict` complains about the array style.

Reply:
I took the non-standard code from https://github.com/InstantClientTap/homebrew-instantclient/blob/master/Formula/instantclient-sqlplus.rb. I'll make the array fix tomorrow. Thanks.

Reply:
Right, I am sorry, the non-standard location code came from another PR, I should have requested then.

Thank you for the fixes!